Riskified.SDK/Model/OrderElements/LineItem.cs

```diff
@@ -29,11 +50,19 @@ public class LineItem : IJsonSerializable
             Condition = condition;
             RequiresShipping = requiresShipping;
             Seller = seller;
+            DeliveredTo = deliveredTo;
+
+            // Events Tickets Industry
+            EventCategoryName = eventCategoryName;
```

Comment:
we changed to category, let's change the name too? 
we're already breaking compatibility (we can change version accordingly)